model/registry/vcs.yaml

```diff
@@ -83,3 +94,18 @@ groups:
           The ID of the change (pull request/merge request) if applicable. This
           is usually a unique (within repository) identifier generated by the VCS system.
         examples: ["123"]
+      - id: vcs.change.state
+        type:
+          members:
```

Comment:
Should there also be a state `closed` for PRs that were closed but not merged?

Reply:
I agree. `closed` would be the other likely value. Neither GItLab nor GitHub identify changes as having been rejected, not accepted, or even approved. (well github does have an approved state, but gitlab doesn't).

For additional states beyond open, merged, closed, I think those would fall under vendor extensions.

Reply:
I added `closed` state in a148a4e86d5ffa54c9298d4c243084e180b22535